sanitize.schema.js

```diff
@@ -0,0 +1,40 @@
+const sanitize = require('hast-util-sanitize/lib/github.json');
```

Comment:
I pulled all of our sanitization schema defaults out of `index.js` in to its own file!